### ISSUE TITLE
OpenMed nemotron fine-tunes run today via PRIVACY_FILTER_MODEL, but verify recall first

### ISSUE THREAD

**Issue by FiveBoroughs** [FiveBoroughs/privacy-filter-local#5]
Three related artifacts, all the same model family:

- [`OpenMed/privacy-filter-nemotron`](https://huggingface.co/OpenMed/privacy-filter-nemotron) — Apache-2.0, a fine-tune of `openai/privacy-filter` on `nvidia/Nemotron-PII`, 2827 MB, 221 BIOES labels over 55 categories
- [`OpenMed/privacy-filter-nemotron-v2`](https://huggingface.co/OpenMed/privacy-filter-nemotron-v2) — same 55 categories, 20 languages including French, "more recall-oriented adaptation recipe"
- [`LocalAI-io/privacy-filter-nemotron-GGUF`](https://huggingface.co/LocalAI-io/privacy-filter-nemotron-GGUF) — GGUF of v1, f16 2817 MB and q8 1638 MB

Unlike #2, #3 and #4, this needs no backend seam. It keeps the `OpenAIPrivacyFilterForTokenClassification` architecture, the same tokenizer, and `max_position_embeddings` of 131072, so no context-limit hazard either. `normalize_label` already strips the `B-`/`I-`/`E-`/`S-` prefixes and the pipeline's `aggregation_strategy="simple"` groups the spans.

## It runs today, unchanged

```
PRIVACY_FILTER_MODEL=OpenMed/privacy-filter-nemotron ./privacy-filter-service ensure
```

`/health` reports `model: OpenMed/privacy-filter-nemotron`, bf16, `max_tokens: 4096`, and scans return the fine-grained labels. No code change, no new dependency. `start-service` already forwards `PRIVACY_FILTER_MODEL`.

## But measure recall before switching

Same four-line synthetic input through both models. Typed labels are a clear gain, and on this sample the fine-tune also caught things the current model missed (`cvv`, a clean `ipv4` instead of three fragments, a clean card number instead of one that swallowed the following word).

It also lost two things that matter more:

| Value in the input | `openai/privacy-filter` | `OpenMed/privacy-filter-nemotron` |
|---|---|---|
| IBAN `FR7630006000011234567890189` | `account_number`, full span | `account_number` on `FR` only |
| Passport `19FR84221` | `account_number` | not detected |
| Licence `B-8827-XQ` | `account_number`, full span | `license_plate` on `-8827-XQ`, drops the `B` |
| CVV `447` | not detected | `cvv` |
| IP `192.0.2.44` | three fragments | `ipv4`, one span |

What that does to `redact` is the point:

```
# nemotron
IBAN [ACCOUNT_NUMBER]7630006000011234567890189.
Passport 19FR84221, driver licence B[LICENSE_PLATE], ...

# current model, same input
survives: 447
```

The fine-tune leaves 25 digits of an IBAN and an entire passport number in the clear. The current model leaks a three-digit CVV. For a redaction tool those are not comparable failures.

This is one short synthetic sample, so it is an anecdote rather than an evaluation, and it is not a reason to reject the model. It is a reason to make recall the acceptance criterion and to run a real corpus both ways before changing the default, which is the same conclusion #4 reached from its benchmark table. Worth testing v2 specifically here, since a more recall-oriented recipe is exactly the axis on which v1 fell short.

## Blockers and notes

- **v2 licensing.** The card declares `license: other`, there is no `LICENSE` file (404), and the card body has no licence section. Terms are unknown, so v2 cannot be adopted or documented as an option until that is resolved. v1 is Apache-2.0 and has no such problem.
- **Label migration.** 55 categories against our current 8 means every `.privacy-filter-reviewed-findings` entry stops matching, since `ReviewedFinding` is keyed on `(path, label, digest)`, and redaction markers change from `[PRIVATE_PERSON]` to `[FIRST_NAME]` and `[LAST_NAME]`. Same migration as #2 and #4.
- **Split names.** The fine-tune emits `first_name` and `last_name` separately where we emit one `private_person`. Adjacent-finding merging in `merge_findings` only joins same-label spans, so a full name now redacts as two markers.
- **GGUF.** Uses a custom architecture, `openai-privacy-filter`, that is not in upstream llama.cpp; it needs `privacy-filter.cpp`, a standalone GGML engine from the same publisher. Attractive for #1's CPU-only, no-Python goal, but it trades a mainstream runtime for a single-purpose one, which deserves its own decision rather than being assumed.

**Comment by FiveBoroughs** [comment]
> *This was generated by AI during triage.*

Add [`OpenMed/privacy-filter-multilingual-v2`](https://huggingface.co/OpenMed/privacy-filter-multilingual-v2) to this issue and make it the first OpenMed variant to benchmark.

It is Apache-2.0 and uses the same `OpenAIPrivacyFilterForTokenClassification` architecture as the current model, so it can be evaluated without adding another backend. It covers 16 languages and 54 categories.

Its raw PIIMB score is strong, but the training data overlaps five of PIIMB's six tasks: AI4Privacy English, AI4Privacy multilingual, Nemotron, Gretel, and Privy. The unseen MAPA task scores 0.373 masking F2. PIIMB is useful screening evidence here, not a held-out result.

Add these acceptance criteria:

- [ ] Benchmark a pinned revision on the local corpus.
- [ ] Report every unmapped label instead of dropping it.
- [ ] Compare it against OpenAI ONNX fp16, OpenAI ONNX int8, GLiNER2, and PPLX.
- [ ] Report recall separately for direct identifiers, quasi-identifiers, and secrets.
- [ ] Record false alarms and disagreements by text.
- [ ] Keep `OpenMed/privacy-filter-nemotron-v2` blocked until its licence is resolved.
- [ ] Do not change the default based on PIIMB results alone.
